### PR TITLE
Target Mode Enum [Target Docs 7/?]

### PR DESCRIPTION
**Pull request by fig02**
The name of "target mode" will likely be changing very soon but thats a later problem. 
This is a find and replace PR except for reorganizing `sTargetRanges` in z_actor.c

### CODE REVIEW

**Review by AngheloAlf — APPROVED** [review]

**Comment by fig02** [comment]
bss is fixed

File: src/overlays/actors/ovl_Bg_Jya_Bombchuiwa/z_bg_jya_bombchuiwa.c [hunk 1]
```diff
@@ -54,10 +54,8 @@ static ColliderJntSphInit sJntSphInit = {
 };
 
 static InitChainEntry sInitChain[] = {
-    ICHAIN_U8(targetMode, 3, ICHAIN_CONTINUE),
-    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
-    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),
-    ICHAIN_F32(uncullZoneScale, 1000, ICHAIN_CONTINUE),
+    ICHAIN_U8(targetMode, TARGET_MODE_3, ICHAIN_CONTINUE), ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),  ICHAIN_F32(uncullZoneScale, 1000, ICHAIN_CONTINUE),
```
Comment (krm01) [inline src/overlays/actors/ovl_Bg_Jya_Bombchuiwa/z_bg_jya_bombchuiwa.c:58]: some newlines got clobbered here, looks like a few other places too?

**Review by krm01 — APPROVED** [review]
lgtm. if you don't care about the ichain whitepsace, i don't either

File: src/overlays/actors/ovl_Bg_Jya_Bombchuiwa/z_bg_jya_bombchuiwa.c (hunk 1 above)
Comment (fig02) [reply src/overlays/actors/ovl_Bg_Jya_Bombchuiwa/z_bg_jya_bombchuiwa.c:58]: this is all the formatters doing. it decides based on line length and i guess the enums making the lines longer made it choose something else

Comment (krm01) [reply src/overlays/actors/ovl_Bg_Jya_Bombchuiwa/z_bg_jya_bombchuiwa.c:58]: the enums making the lines longer made it choose to make the lines even longer by combining lines? that's.... welp, nevermind then!

**Pull request merged**